### ISSUE TITLE
Test for kinetic release

### ISSUE THREAD

**Issue by chris-smith** [RethinkRobotics-opensource/genjs#13]
We should release a new version of this into kinetic (`rosnodejs` npm release is held up by it) but we need to do more testing first. Mostly, we need to do the following

- [x] Create a workspace A with `gennodejs` and messages, build, check that devel space messages work in `rosnodejs` (probably just running tests, examples)
- [x] Test that workspace A install space messages work in `rosnodejs`
- [x] Create a workspace B overlaying workspace A with new messages, check that devel space messages work in `rosnodejs`
- [x] Test that workspace B install space messages worko
- [ ] Verify each of these again using `Python3`

This depends on merging #12

**Comment by chris-smith** [comment]
@IanTheEngineer and I tested these yesterday - python3 is apparently not supported by ROS and we couldn't get past some lower level `catkin` bug when we tried to use it. We tested against `catkin` and `catkin tools`.

This should be ready to merge into `kinetic`!
